src/cljs/rems/administration/status_flags.cljs

```diff
@@ -48,6 +47,9 @@
    [:label.form-check-label {:for "display-old"}
     (text :t.administration/display-old)]])
 
+(defn active? [item]
+  (and (not (:expired item)) (:enabled item) (not (:archived item))))
```

Comment:
:expired will probably slowly disappear. it's already gone from licenses. but that doesn't matter: a missing :expired is the same as false